Main/ClientApp/src/Actions/CollectionActions.ts

```diff
@@ -49,6 +49,40 @@ export async function loadSharedCollections() {
   });
 }
 
+export async function loadContributorsSharedCollections() {
```

Comment:
The same (of)

Reply:
the problem is that we are receiving shared collections, not users (contributors)